cmd/lotus/eudico.go

```diff
@@ -0,0 +1,253 @@
+package main
+
+import (
+	"context"
```

Comment:
Imported packages are not sorted. We use the following policy: native, external, filecoin-related

Reply:
IIRC, doing `make gen` should get this done for you automatically (FYI in case it helps).

Reply:
Thanks for the heads up. I fixed them now.